src/main-process/atom-application.coffee

```diff
@@ -812,3 +814,10 @@ class AtomApplication
       args.push("--resource-path=#{@resourcePath}")
     app.relaunch({args})
     app.quit()
+
+  clone = (obj) ->
```

Comment:
Maybe use [`Object.assign`](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Object/assign) or [`_.clone`](https://lodash.com/docs/4.17.4#clone) instead of reinventing clone? :)